src/amber/support/dsls.cr

```diff
@@ -23,4 +23,24 @@ module Amber::Support::DSL
 
     {% end %}
   end
+
+  module ControllerActions
+    macro render_both(filename, layout)
```

Comment:
but what is this `content` variable?
why render the file, then the layout? (it seems that I'm missing sth here..)